keps/sig-release/3720-freezing-k8s-gcr-io/README.md

```diff
@@ -0,0 +1,172 @@
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+The Kubernetes project has created and runs the `registry.k8s.io` image registry which is fully controlled by the community.
+This registry has been [GA for several months](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/) now and we need to freeze the old image registry.
+
+## Motivation
+
+Running public image registries is very expensive and eats up a significant chunk of the project's Infrastructure budget. We built `registry.k8s.io` image registry to serve images from various origins around the world depending on the location of the user. For example, an a kops Kubernetes cluster in eu-west-1 can pull images from an AWS S3 bucket in the same region which is very fast and more importantly very cheap for the Kubernetes project.
+
+There was a plan to redirect `k8s.gcr.io` to `registry.k8s.io` but it didn't [work out](https://kubernetes.slack.com/archives/CCK68P2Q2/p1666725317568709) so we backported the image registry defaults to 1.24, 1.23, and 1.22 so all the patch releases from December 2022 will using the new registry by default.
+
+We are currently exceeding our budget as it will take quite a while for end users to upgrade Kubernetes to v1.25 so we want to incentivise our end users to move to the new registry as fast as possible by freezing the registry by 1.27. This would mean that all subsequent image releases will not be available on the old registry.
+
+### Goals
+
+Freeze `k8s.gcr.io` image registry and push all new images and tags exclusively to the `registry.k8s.io` image registry.
+
+### Non-Goals
+
+- `registry.k8s.io` internal implementations details. That is handled separately by sig-k8s-infra.
+
+## Proposal
+
+Freeze the `k8s.gcr.io` image by not pushing any new digests or tags after 1.27 release. The 1.27 release itself won't be available on `k8s.gcr.io`.
+
+I'm proposing that on the 1st of April 2023 (10 days before 1.27 is released):
+
+- `k8s.gcr.io` is frozen and no new images will be published by any subproject.
+- last 1.23 release on `k8s.gcr.io` will be 1.23.18 (goes EoL before the freeze)
+- last 1.24 release on `k8s.gcr.io` will be 1.24.12
+- last 1.25 release on `k8s.gcr.io` will be 1.25.8
+- last 1.26 release on `k8s.gcr.io` will be 1.26.3
+- 1.27.0 will not be available `k8s.gcr.io`
+
+### Risks and Mitigations
+
+There are no risks. The old registry will still be available and you can pull the images before 1.27 on there. This change will also
+affect other users of k8s.gcr.io who should have already updated their helm charts and manifests to use the new registry.
+
+## Design Details
+
+The image promotion process is described [here](https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io). Please read it for full details.
+
+This is the planned technical changes(grouped by repos):
+
+- k-sigs/promo-tools
+  - merge https://github.com/kubernetes-sigs/promo-tools/pull/669
```

Comment:
Is this enough to ensure we can open PRs against `registry.k8s.io` path during a release, specifically the image promotion step?

Reply:
https://calendar.google.com/calendar/u/0/embed?src=agst.us_b07popf7t4avmt4km7eq5tk5ao@group.calendar.google.com

If you look at the calendar, 1.27.rc0 will be cut on the 21st of March and 1.27.rc1 will be cut on the 4th of April. I'm planning on executing the cutover between those two releases so all future manifest changes will be under the registry.k8s.io folder.